cli/cmd/uninstall.go

```diff
@@ -36,9 +36,16 @@ var uninstallCmd = &cobra.Command{
 		}
 		ctx := cmd.Context()
 
+		errMessage := fmt.Errorf("Odigos is not installed in the cluster")
 		ns, err := resources.GetOdigosNamespace(client, ctx)
 		if err != nil {
-			ns = consts.DefaultNamespace
+			if errors.IsNotFound(err) {
+				fmt.Println(errMessage)
+			} else {
+				ns = consts.DefaultNamespace
+				fmt.Println(err)
```

Comment:
What will be printed in this case? will it include a full-blown stack trace?
can you please provide an example for the output?

I suggest at least adding some text that target the user with instructions, like "An error occurred during odigos uninstall. Please check your kubeconfig and try again. If the problem persists or you need assistance, don't hesitate to open an issue or reach out in odigos slack channel"

Reply:
- We can do that, but my concern is that, is this message ```Please check your kubeconfig and try again.```,  generic all types of problem ?